x-pack/plugins/index_management/public/application/sections/home/data_stream_list/data_stream_list.tsx

```diff
@@ -202,6 +206,16 @@ export const DataStreamList: React.FunctionComponent<RouteComponentProps<MatchPa
               </EuiFlexItem>
             </EuiFlexGroup>
           </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiSwitch
+              label={i18n.translate('xpack.idxMgmt.dataStreamListControls.viewManagedSwitchLabel', {
+                defaultMessage: 'View managed',
```

Comment:
<img width="448" alt="Screen Shot 2020-11-12 at 1 03 02 PM" src="https://user-images.githubusercontent.com/5226211/98979689-a3432a80-24e9-11eb-96a7-50af35919456.png">

WDYT about changing this to "Include managed" to align with the other toggle?